### DISCUSSION TITLE
How to test text inside the slide.

### DISCUSSION THREAD

**Discussion by nicholas0983** [cypress-io/cypress#14705] [Questions and Help] [answered]
Hi guys, sorry for asking for help,
So, I'm trying to find some document to understand how I can test slide text.
I have a slide, I would like to test the texts on 3 pages of the slide.

 cy.get("[data-cy='bannerContentText']").contains('About us text.', { multiple: true });

I dont now how can start this.
Sorry about my amateur doubt.
**Script Slide** 
`
<pre class="line-numbers">
  <code class="language-css">
<script>
export default {
  data() {
    return {
      slides: [
        {
          url: "/about-us",
          text: "About us text.",
          img: "/carousel/about-us.jpg",
        },
        {
          url: "/transparency",
          text: "Transparency text.",
          img: "/carousel/transparency.jpg",
        },
        {
          url: "/cars-projects",
          text: "Projects.",
          img: "/carousel/cars-projects.jpg",
        },
      ],
    };
  },
};
</script>
  </code>
</pre>
`
**Cyporess Test**

`
<pre class="line-numbers">
  <code class="language-css">
    describe("Home page", () => {
      it("Checking titles", () => {
     	 cy.visit("/") 
     	 cy.get("[data-cy='bannerContentText']").contains('About us text.', { multiple: true });
      // cy.get("[data-cy='bannerContentText']").eq(0)('contains', "Transparency text.")
      })
    })
  </code>
</pre>
`

**Comment by bahmutov** [comment, marked as answer]
If you just want to confirm there is this text in the banner, use [cy.contains](https://on.cypress.io/contains) command. It allows you to even specify the selector first.

```js
cy.contains("[data-cy='bannerContentText']", 'About us text.')
```

I would also recommend watching the free Cypress courses at https://on.cypress.io/courses to see the basics of Cypress tests

**Reply by nicholas0983** [reply]
Hey Bahmutov, thank you so much for the help.
I did like this and it worked perfectly.

    cy.visit("/")
    cy.get(".v-carousel__controls button").eq(0).click()
    cy.get("[data-cy='bannerContentText']").contains("Text1.")
    cy.get(".v-carousel__controls button").eq(1).click()
    cy.get("[data-cy='bannerContentText']").contains("Text2")
    cy.get(".v-carousel__controls button").eq(2).click()
    cy.get("[data-cy='bannerContentText']").contains("Text3")
